### PR TITLE
Issue 729 test Team Destroy

### PR DESCRIPTION
**Pull request by santiaago**
issue 728 should be merged first

### CODE REVIEW

File: models/team_test.go [hunk 1]
```diff
@@ -95,3 +158,11 @@ func checkTeamInvertedIndex(t *testing.T, c aetest.Context, got *Team, want test
 
 	return errors.New("team not found")
 }
+
+// errString returns the string representation of an error.
+func errString(err error) string {
```
Comment (rjourde) [inline models/team_test.go]: could you use the same function name than in user_test ?

File: models/team_test.go [hunk 2]
```diff
@@ -58,6 +59,68 @@ func TestCreateTeam(t *testing.T) {
...
+			overrideId: true,
+			newId:      11,
+			err:        "Cannot find team with Id",
+		},
+	}
+
+	for i, test := range tests {
+		t.Log(test.title)
+		var got *Team
+		if got, err = CreateTeam(c, test.team.name, test.team.description, test.team.adminId, test.team.private); err != nil {
```
Comment (rjourde) [inline models/team_test.go]: Maybe it is better to do first the creation of the team then define the testing structure then test first "cannot find" test case then the "can destroy". What do you think ?

**Comment by santiaago** [comment]
They are different entities so it should not matter the order anyway

File: models/team_test.go (hunk 1 above)
Comment (santiaago) [inline models/team_test.go]: yes I saw this, I will refactor them in another issue, maybe even create a testing package :)

**Comment by rjourde** [comment]
But you are creating twice the same team, right ? So if you create only one team in that case the order will matter.

**Comment by santiaago** [comment]
In that case I will change the Id numbers to be different

**Comment by rjourde** [comment]
ok.

**Comment by santiaago** [comment]
it is another entity, the number that you see is the adminId so that is irrelevant.

**Pull request merged**